src/dadbot.js

```diff
@@ -76,12 +76,42 @@ module.exports = (robot) => {
                 str.substr(1, str.length - 1).toLowerCase();
         };
 
-        let firstWord = capitalize(res.match[1]);
-        let secondWord = capitalize(res.match[2]);
+        let lowerCase = (str) => {
+            if (str == undefined) return undefined;
+            return str.toLowerCase();
+        };
+
+        let isIntensifier = (str) =>
+            intensifiers.indexOf(str.toLowerCase()) >= 0;
+
+        let firstWord = lowerCase(res.match[1]);
+        let secondWord = lowerCase(res.match[2]);
+
+        // caught something like "i am very very tired"
+        if (isIntensifier(firstWord) && firstWord == secondWord) {
```

Comment:
What if, instead of just looking for duplicate first and second words, you keep look at if the second word is an intensifier as well?

For example, it would then handle the case where two intensifiers (that are different) come in a row:

`"I am so extremely tired"` --> `"Hi Mr(s) Tired, can I call you So Extremely?"

I think right now as implemented, this would still output:
`"I am so extremely tired"` --> `"Hi Mr(s) Extremely, can I call you So?"`